spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/AdminConfigTests.java

```diff
@@ -53,11 +62,36 @@
 	private KafkaMessageChannelBinder binder;
 
 	@Test
-	public void testProps() {
-		KafkaConsumerProperties consumerProps = this.binder.getExtendedConsumerProperties("input");
-		KafkaAdminProperties admin = consumerProps.getAdmin();
+	public void testDeprecatedAdminConfigurationToMapTopicProperties() {
+		final KafkaConsumerProperties consumerProps = this.binder.getExtendedConsumerProperties("input");
+		final KafkaAdminProperties admin = consumerProps.getAdmin();
```

Comment:
Once it's a subclass, we can get rid of all test references to the deprecated `getAdmin()` methods (we can still leave setting them via the old property names).

Reply:
done!